packages/ui/src/components/Checkbox/Checkbox.test.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import { Checkbox } from './Checkbox';
+
+describe('Checkbox component', () => {
+  it('renders correctly', () => {
+    render(<Checkbox value="test" onChange={() => {}} checked aria-label="checkbox" />);
+
+    const checkbox = screen.getByLabelText('checkbox');
+
+    expect(checkbox).toBeInTheDocument();
+  });
+  it('checks that checked checkbox is checked', () => {
+    render(<Checkbox value="test" onChange={() => {}} checked aria-label="checkbox" />);
+
+    const checkbox = screen.getByLabelText('checkbox');
+
+    expect(checkbox).toBeChecked();
+  });
+  it('checks checkbox displays correct labels', () => {
+    const CbLabel = 'checkbox name';
```

Comment:
As a good practice, only classes, components, and types should be written CamelCase. It would be better to change it to just `label`